hack/make-bundle-vars.mk

```diff
@@ -0,0 +1,6 @@
+
+MCG_CONSOLE_IMG_NAME ?= mcg-ms-console
+MCG_CONSOLE_IMG_TAG ?= latest
+MCG_CONSOLE_IMG_LOCATION ?= quay.io/skatiyar
+MCG_CONSOLE_IMG ?= $(MCG_CONSOLE_IMG_LOCATION)/$(MCG_CONSOLE_IMG_NAME):$(MCG_CONSOLE_IMG_TAG)
```

Comment:
Can we move them to `.env` (since these are env vars)? Any reason not to?

Reply:
These are not env vars but the tag names used in main Make file. This way of defining tags in seperate file will leave main Makefile clean

Reply:
I misspoke, not env vars, just vars. But what I meant was, these are defined and used in the same way as https://github.com/red-hat-storage/mcg-osd-deployer/blob/main/shim/.env, are they not?

Reply:
Its more of format. Even if you check https://github.com/red-hat-storage/odf-operator/blob/main/hack/make-bundle-vars.mk, where Makefile vars/tags are part of the similar file. `.mk` is one such file format used to have such parameters.